### ISSUE TITLE
您好，请问IAT与yolov3联合训练的时候，loss函数中似乎没有IAT的损失函数 ，只有yolov3的

### ISSUE THREAD

**Issue by lissongod** [cuiziteng/Illumination-Adaptive-Transformer#72]
您好，请问IAT与yolov3联合训练的时候，loss函数中似乎没有IAT的损失函数 ，只有yolov3的

**Comment by cuiziteng** [comment]
Yes, we directly add the perception loss function, because only the detection loss is enough, the object detection task is concentrate on detection performance rather than image quality.

In fact the follow-up work also do not add image reconstruction loss function, like FeatEnhancer:
https://openaccess.thecvf.com/content/ICCV2023/papers/Hashmi_FeatEnHancer_Enhancing_Hierarchical_Features_for_Object_Detection_and_Beyond_Under_ICCV_2023_paper.pdf
